source/slides/13_ansible.rst

```diff
@@ -0,0 +1,520 @@
+.. _13_ansible:
+
+Ansible
+=======
+
+What is Ansible?
+----------------
+
+* Started in 2012
+* More than 600 contributors
+* Orchestration Engine
+
+  * Configuration Management
+  * Application Deployment
+  * Continuous Delivery
+
+Ansible Commands
+----------------
+
+.. rst-class:: build
+
+``ansible``
+  Low level command to execute simple Ansible resources
+``ansible-playbook``
+  Tool to run Ansible Playbooks
+``ansible-pull``
+  Tool to pull remote Ansible configurations and run the playbooks
+``ansible-doc``
+  Tool to show documentation on Ansible modules
+``ansible-galaxy``
+  Tool to manage roles from Ansible Galaxy, a shared repository of Ansible roles
+  maintained by the community
+
+Ansible Components
+------------------
+
+.. rst-class:: build
+
+**Modules**
+  Python-driven code that actually does the work in Ansible and executed on the
+  remote host.
+**Inventory**
+  Defines how Ansible with interact with remote hosts and defines hosts, groups
+  of hosts in a logical manner.
+**Playbooks**
+  Ansible's configuration, deployment and orchestration language that uses the
+  YAML format.
+**Roles**
+  Reusable list of tasks and playbooks typically centered in a specific task.
+
+Modules
+-------
+
+* Python code that is executed on the remote host
+* Always returns JSON data
+* Controls system resources, executing system commands
```

Comment:
Modules sometimes execute system commands; sometimes they use local library bindings to accomplish the configured task.  I would probably amend it to say:

    Control and manage system resources